### ISSUE TITLE
CLI doesnt work

### ISSUE THREAD

**Issue by Joshswooft** [JRJurman/excalidraw-to-svg#3]
I pulled down your repo and tried to run the command but it fails both on node version 16 and 18. 
MacOS: v12.6
Chip: Apple M1 Pro

<img width="664" alt="Screenshot 2023-03-16 at 18 14 32" src="https://user-images.githubusercontent.com/60711758/225714775-c2493108-4a72-4192-8445-e99c4b4538da.png">

using the example drawing you provided i.e. 

```{
  "type": "excalidraw",
  "version": 2,
  "source": "https://excalidraw.com",
  "elements": [
    {
      "id": "vWrqOAfkind2qcm7LDAGZ",
      "type": "ellipse",
      "x": 414,
      "y": 237,
      "width": 214,
      "height": 214,
      "angle": 0,
      "strokeColor": "#000000",
      "backgroundColor": "#15aabf",
      "fillStyle": "hachure",
      "strokeWidth": 1,
      "strokeStyle": "solid",
      "roughness": 1,
      "opacity": 100,
      "groupIds": [],
      "strokeSharpness": "sharp",
      "seed": 1041657908,
      "version": 120,
      "versionNonce": 1188004276,
      "isDeleted": false,
      "boundElementIds": null
    }
  ],
  "appState": {
    "viewBackgroundColor": "#ffffff",
    "gridSize": null
  }
}
```

**Comment by JRJurman** [comment]
Sorry about that! This should be fixed now (with version v3.0.1) - let me know if there are still issues! 👍

**Comment by Joshswooft** [comment]
@JRJurman that's working now, thanks!
